src/main/java/seedu/duke/logic/commands/favcommand/ClearFavCommand.java

```diff
@@ -0,0 +1,12 @@
+package seedu.duke.logic.commands.favcommand;
+
+import seedu.duke.favorite.FavList;
+import seedu.duke.logic.commands.Command;
+
+public class ClearFavCommand extends Command {
+
+    @Override
+    public void executeCommand() {
+        FavList.clearFav();
+    }
```

Comment:
Would it be better to call the Ui method to print the message here instead?

Reply:
Can, but I was following the structure from ethan's skeleton code, the methods are all in FavList

Reply:
I see, okie!